pyproject.toml

```diff
@@ -24,6 +24,7 @@ dependencies = [
     "scikit-image >= 0.24.0",
     "reciprocalspaceship >= 1.0.2",
     "structlog >= 24.4.0",
+    "setuptools >= 75.5.0",
```

Comment:
I had to do this to get `rs` to install properly in our CI. When you generate a new `conda` environment, it seems like you get `setuptools` for free, but in GHA installing with just `pip`, that's not the case